dash-spv-coinjoin/src/messages/coinjoin_queue_message.rs

```diff
@@ -89,6 +91,7 @@ impl encode::Decodable for CoinJoinQueueMessage {
         let pro_tx_hash = UInt256::consensus_decode(&mut d)?;
         let time = i64::consensus_decode(&mut d)?;
         let ready: bool = bool::consensus_decode(&mut d)?;
+        let _signature_len = VarInt::consensus_decode(&mut d)?;
         let signature: Option<UInt768> = UInt768::consensus_decode(&mut d).ok();
```

Comment:
The issue with signature verification was that the message wasn't parsed correctly (there is the length before the signature)